vignettes/generation.Rmd

```diff
@@ -0,0 +1,170 @@
+
+For the example here, we use the example data in the `MEDIPSData` package. This contains pairs of tumour and normal tissue from the same patients, with MEDIP-seq performed as the enrichment step. These bam files are downsampled to only contain chromosomes 20-22, and are aligned to hg19. 
+Note, we highly recommend using the hg38/GRCh38 reference genome rather than hg19 as used here!
+
+```{r, eval = requireNamespace("MEDIPSData")}
+sampleTable <- tribble(~sample_name, ~group, ~tumour, ~patient, ~file_name,
+    "Normal1", "Normal1", "Normal", "Patient1", system.file("extdata", "NSCLC_MeDIP_1N_fst_chr_20_21_22.bam", package = "MEDIPSData", mustWork = TRUE),
+    "Normal2", "Normal1", "Normal", "Patient2", system.file("extdata", "NSCLC_MeDIP_2N_fst_chr_20_21_22.bam", package = "MEDIPSData", mustWork = TRUE),
+    "Tumour1", "Tumour1", "Tumour", "Patient1", system.file("extdata", "NSCLC_MeDIP_1T_fst_chr_20_21_22.bam", package = "MEDIPSData", mustWork = TRUE),
+    "Tumour2", "Tumour2", "Tumour", "Patient2", system.file("extdata", "NSCLC_MeDIP_2T_fst_chr_20_21_22.bam", package = "MEDIPSData", mustWork = TRUE)
+    )
+```
+
+## Making a qseaSet
+The `qsea` tutorial details the steps involved in generating a `qseaSet`. 
+We provide a function, `makeQset`, which wraps these up into one function, although you may still use the rest of the `mesa` functionality with or without using this to create your `qseaSet`.
+
+Before creating the `qseaSet`, you need to have:
+
+* The `sampleTable` of metadata, including the file paths to the bam files.
+* The genome to which the bam files have been mapped, and the corresponding `BSgenome` package name, and the package to be installed. You can check what BSgenome packages are available on BioConductor via `BSgenome::available.genomes()`, or make your own if necessary (`vignette("BSgenomeForge", package = "BSgenome")`).
+* Whether the aligned reads are paired-end sequencing or not. If single-end sequencing has been used, you need to know the expected DNA fragment length from an experimental measurement, to extend all the reads to this length.
+* An expected fragment length and SD of your DNA fragment size distribution. There are a range of tools that can be used to visualise this on your bam files, for instance [QualiMap](http://qualimap.conesalab.org/), but a rough ballpark should also be experimentally known based on how the DNA material has been processed for sequencing. 
+* What window size to use. In making a `qseaSet`, the genome is split into fixed windows with equal size, and all analysis is performed on these windows as independent units. We generally use 300bp windows, as that is a little bigger than our average DNA fragment sizes, which are in the 150-250bp range. If the DNA is particularly long fragments, a larger bin size may make sense.
```

Comment:
Would such windows also work for low-depth samples? Since it's mentioned earlier, I think it should be pointed out if not.